cli/tests/integrations/test_package.py

```diff
@@ -323,45 +323,38 @@ def test_bad_install():
 
 
 def test_bad_install_helloworld_msg():
-    stdout = (
+    stdout2 = (
```

Comment:
maybe call it `version` instead of stdout2?

Reply:
I've renamed it to `terms_conditions` as it mainly contains this.